packages/vscode-extension/src/webview/providers/LaunchConfigProvider.tsx

```diff
@@ -63,9 +76,22 @@ export default function LaunchConfigProvider({ children }: PropsWithChildren) {
     [config, setConfig]
   );
 
+  const addCustomApplicationRoot = (appRoot: string) => {
+    const newState = [...applicationRoots, appRoot];
+    setApplicationRoots(newState);
+    launchConfig.addCustomApplicationRoot(appRoot);
```

Comment:
Could we just do:
```suggestion
    launchConfig.addCustomApplicationRoot(appRoot);
```
and rely on the `launchConfig` listener to update it for us?

Reply:
yes, thank you